todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/taskdetail/TaskDetailFragment.java

```diff
@@ -68,13 +89,60 @@ public static TaskDetailFragment newInstance(String taskId) {
     @Override
     public void onResume() {
         super.onResume();
-        mPresenter.subscribe();
+        bind();
     }
 
     @Override
     public void onPause() {
+        unbind();
         super.onPause();
-        mPresenter.unsubscribe();
+    }
+
+    private void bind() {
+        mSubscription = new CompositeSubscription();
```

Comment:
rxjava is your binding implementation. when you need to change that it will affect all the views.
this snippet is your declaration .  setLoadingIndicator(active); is part of this binding declaration, showMissingTask(); also. 
Also are you really going to have a single binding like this per V. If you have 10 items to observe this will not look nice, sometimes there could be 100. My point is the view needs to be clean. Fragments in android are already pretty nasty to handle on its own anyway :)